src/__test__/missionReducer.test.js

```diff
@@ -1,4 +1,4 @@
-import missionReducer, { missionReserve } from '../../Redux/Missions/MissionReducer';
+import missionReducer, { missionReserve } from '../Redux/Missions/MissionReducer';
 
```

Comment:
Please ensure that when a user clicks the "Leave Mission" button, action needs to be dispatched to update the store.

Below is the error I get when I click on the `Leave Mission`. Kindly fix that error.

![image](https://user-images.githubusercontent.com/10332499/159091667-d8750b1a-05fc-4fc6-8047-324cc3b9184d.png)